lib/sidekiq/grouping/actor.rb

```diff
@@ -9,23 +9,53 @@ def initialize
       end
 
       private
-      
+
       def start_polling
+        return unless Sidekiq::Grouping::Config.enabled
```

Comment:
The class Actor is a Celluloid actor or cell that lives in its own thread. Here it's used to read for batches to flush every x seconds. 

I added the `enabled` option to the GEM configuration to disable the event to avoid calling Responsys in environments we don't want to (tests + staging servers). This is what this line is for.

I just realized that it could even be done before the actor gets registered to Sidekiq in `link_to_sidekiq-manager`. I'll move the line down there.